build-files/abricate/1.2.0/README.md

```diff
@@ -0,0 +1,52 @@
+# ABRicate
+
+> ABRicate: Mass screening of contigs for antimicrobial resistance or virulence genes. It comes bundled with multiple databases: NCBI, CARD, ARG-ANNOT, Resfinder, MEGARES, EcOH, PlasmidFinder, Ecoli_VF and VFDB.
+
+Main tool : [ABRicate](https://github.com/tseemann/abricate)
+
+Additional tools:
+
+- any2fasta 0.8.1
+- ncbi-blast+ 2.12.0
+- perl 5.38.2
+- bioperl 1.7.8
+
+Additional information:
+
+This image uses the most up-to-date databases at the time of building. This image is rebuilt every month on Dockerhub and Quay.io with the tag `${abricate version}-${data image was deployed}` as well as `latest`.
+
+
+## Example Usage
+
+```bash
+# list out the available databases
+$ abricate --list
+DATABASE        SEQUENCES       DBTYPE  DATE
+victors 4545    nucl    2025-Dec-30
+resfinder       3206    nucl    2025-Dec-30
+card    6052    nucl    2025-Dec-30
+megares 6635    nucl    2025-Dec-30
+ecoli_vf        2701    nucl    2025-Dec-30
+vfdb    4592    nucl    2025-Dec-30
+ecoh    597     nucl    2025-Dec-30
+bacmet2 746     prot    2025-Dec-30
+argannot        2224    nucl    2025-Dec-30
+plasmidfinder   488     nucl    2025-Dec-30
+ncbi    8035    nucl    2025-Dec-30
```

Comment:
Depending on what day this docker image is deployed, we will need to update these lines to match the date